bin/bump-version

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 -p [major | minor | patch]" 1>&2; exit 1; }
+
+while getopts "p:" o; do
+  case "${o}" in
+    p)
+      patch_level=${OPTARG}
+      (( patch_level == 'major' || patch_level == 'minor' || patch_level == 'patch'))
+      ;;
+    *)
+      usage
+      ;;
+  esac
+done
+
+echo "$patch_level"
+
+if [[ -z "${patch_level}" ]]; then
+  usage
+fi
+
+new_version=$(npm version "${patch_level}" --no-git-tag-version)
+git checkout -b "${new_version}"-release-notes
```

Comment:
should we first checkout/fetch main?

Reply:
That's a good question, I went back and forth on that.

I decided to punt on it on the assuming that the user could be releasing from a `release/vX` branch in future and if something went wrong with the checkout/fetch we'd have to be careful to catch it which felt like scope creep.